src/aiortc/codecs/vpx.py

```diff
@@ -283,10 +300,23 @@ def encode(
             self.cfg.rc_buf_sz = 1000
             self.cfg.kf_mode = lib.VPX_KF_AUTO
             self.cfg.kf_max_dist = 3000
+            self.vpx_min_bitrate = MIN_BITRATE
+            self.vpx_max_bitrate = MAX_BITRATE
+            if quantizer < 0 and not enable_gcc and MIN_BITRATE != MAX_BITRATE:
```

Comment:
Is this because you want to keep the environment variable pipeline still?

Reply:
yes